package/kernel/linux/modules/block.mk

```diff
@@ -27,10 +27,9 @@ define KernelPackage/ata-core
   TITLE:=Serial and Parallel ATA support
   DEPENDS:=@PCI_SUPPORT||TARGET_sunxi +kmod-scsi-core
   KCONFIG:=CONFIG_ATA
-  FILES:=$(LINUX_DIR)/drivers/ata/libata.ko
-ifneq ($(wildcard $(LINUX_DIR)/drivers/ata/libahci.ko),)
-  FILES+=$(LINUX_DIR)/drivers/ata/libahci.ko
-endif
+  FILES:= \
+	$(LINUX_DIR)/drivers/ata/libata.ko \
+	-$(LINUX_DIR)/drivers/ata/libahci.ko
```

Comment:
Guess I should mention, I'm pretty sure this one is no longer optional.  Just I was merely trying to adjust the mechanism and not checking for whether this still made sense.

Reply:
Hmm, nevermind.  On x86 this is non-optional, but other architectures sometimes omit.